src/main/java/pages/HomePage.java

```diff
@@ -11,58 +10,88 @@
 			passwordTextCss = helper.parseJSONToString("passwordTextCss", filePathCommonOR),
-			submitButtonClassName = helper.parseJSONToString("submitButtonClassName", filePathCommonOR),
+			loginSubmitButtonClassName = helper.parseJSONToString("submitButtonClassName", filePathCommonOR),
 			dealsClassName = helper.parseJSONToString("dealsLinksClassName", filePathHomePage),
 			corporateEnquiryFormLabelId = helper.parseJSONToString("corporateEnquiryFormLabelId", filePathHomePage),
 			hotelLinksClassName = helper.parseJSONToString("hotelLinksClassName", filePathHomePage),
 			megadropdownClassName = helper.parseJSONToString("megadropdownClassName", filePathHomePage),
-			corporateNameId=helper.parseJSONToString("corporateNameId", filePathHomePage),
-			corporatePhoneId=helper.parseJSONToString("corporatePhoneId", filePathHomePage),
-			corporateEmailId=helper.parseJSONToString("corporateEmailId", filePathHomePage),
-			corporateEnquiryId=helper.parseJSONToString("corporateEnquiryId", filePathHomePage),
-			corporateEnquirySubmitButtonId=helper.parseJSONToString("corporateEnquirySubmitButtonId", filePathHomePage),
-			corporateEnquirySuccessId=helper.parseJSONToString("corporateEnquirySuccessId", filePathHomePage);
-	
+			corporateNameId = helper.parseJSONToString("corporateNameId", filePathHomePage),
+			corporatePhoneId = helper.parseJSONToString("corporatePhoneId", filePathHomePage),
+			corporateEmailId = helper.parseJSONToString("corporateEmailId", filePathHomePage),
+			corporateEnquiryId = helper.parseJSONToString("corporateEnquiryId", filePathHomePage),
+			corporateEnquirySubmitButtonId = helper.parseJSONToString("corporateEnquirySubmitButtonId",
+					filePathHomePage),
+			corporateEnquirySuccessId = helper.parseJSONToString("corporateEnquirySuccessId", filePathHomePage),
+			blankLoginValidationMessageId = helper.parseJSONToString("blankLoginValidationMessageId", filePathCommonOR),
+			megaDDLink = helper.parseJSONToString("megaDDLinkText", filePathHomePage),
+			viewAllLinkText = helper.parseJSONToString("viewAllLinkText", filePathHomePage),
+			allCitiesLinkText = helper.parseJSONToString("allCitiesLinkText", filePathHomePage),
+			logoutLinkText =helper.parseJSONToString("logoutLinkText",filePathCommonOR );
 
 	public void isValid() {
 		validateHeaderElements();
 		validateFooterElements();
 		Assert.assertTrue(helper.findElementByClassName(dealsClassName).isDisplayed());
 		Assert.assertTrue(helper.findElementById(corporateEnquiryFormLabelId).isDisplayed());
+		Assert.assertTrue(helper.findElementByClassName(megadropdownClassName).isDisplayed());
 		List<WebElement> links = helper.findElementsByClassName(hotelLinksClassName);
  		for (int i = 0; i < links.size(); i++) {
 			Assert.assertTrue(links.get(i).isEnabled());
 		}
-		Assert.assertTrue(helper.findElementByClassName(megadropdownClassName).isDisplayed());
 	}
 
+	public String getBlankLoginValidationMessage() {
```

Comment:
Suggestion: we should use some Top level method comments. It will help us to identify the use of all methods.